lib/beaker/hypervisor/vagrant.rb

```diff
@@ -160,6 +160,17 @@ def cleanup
       FileUtils.rm_rf(@vagrant_path)
     end
 
+    #snapshotting depends on https://github.com/scalefactory/vagrant-multiprovider-snap
+    def take_snapshot(host,snapshot_name)
+      @logger.debug "Creating snapshot of #{host}"
```

Comment:
I'm calling take_snapshot from host.rb with the name.  I'm not passing the Host.
https://github.com/puppetlabs/beaker/pull/654/files#diff-25c9ddfb41958c160da1d3b679793ddaR382

Would it be better style to do "self[:hypervisor].take_snapshot(self, snapshot_name)" and call host.name in vagrant.rb as you've suggested?